### PR TITLE
Replace bcftools/tabix by tabix/tabix

### PR DESCRIPTION
**Pull request by MaxUlysse**

### CODE REVIEW

**Comment by MaxUlysse** [comment]
I'm assuming here that the input is a bgzip vcf file.
I'm wondering if we should consider that input could be just a vcf too, and in this case, we would need to add a bgzip command to compress it.
In any case, the vcf file need to be sorted for indexing (whether it is gzip or not).

**Comment by MaxUlysse** [comment]
And actually, just realized that we can also tabix index bed files, so will need to include that in current version too

**Comment by MaxUlysse** [comment]
ok, so I made HTSLIB_TABIX more modular, works for bed, gff and vcf files.
All files need to be bgziped before, so added a bgziped bed and gff.
(I assumed the A.bed.gz was gunziped and not bgziped, so prefered to add a B.bed.gz).
I'll add a HTSLIB_BGZIP process later that can work with this module to start from regular bed, gff and vcf files.
I did not include the tabix indexing of sam as I assume no one is using that anymore.

**Comment by MaxUlysse** [comment]
Another point of interest that could be good to address.
I'm not using meta here, mainly because even if I do tabix index all my vcf files within Sarek (that would have such information), I also use that to index some reference files that don't have it.
So I'm wondering on how to asses that matter.

**Comment by MaxUlysse** [comment]
cf https://github.com/nf-core/modules/pull/134

**Comment by MaxUlysse** [comment]
Following the conversation on Slack:

- Rename HTSLIB/TABIX into TABIX/TABIX

- Merge BCFTOOLS/TABIX with it

- use the TABIX container

File: tests/software/tabix/tabix/main.nf [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { TABIX_TABIX as TABIX_BED } from '../../../../software/tabix/tabix/main.nf' addParams( options: ['args': '-p bed'] )
+include { TABIX_TABIX as TABIX_GFF } from '../../../../software/tabix/tabix/main.nf' addParams( options: ['args': '-p gff'] )
+include { TABIX_TABIX as TABIX_VCF } from '../../../../software/tabix/tabix/main.nf' addParams( options: ['args': '-p vcf'] )
```
Comment (drpatelh) [inline tests/software/tabix/tabix/main.nf:7]: Love these different tests ❤️

File: .github/filters.yml [hunk 2]
```diff
@@ -136,6 +132,10 @@ gunzip:
   - software/gunzip/**
   - tests/software/gunzip/**
 
+htslib_tabix:
+  - software/htslib/tabix/**
+  - tests/software/htslib/tabix/**
```
Comment (drpatelh) [inline .github/filters.yml:137]:
```suggestion
tabix_tabix:
  - software/tabix/tabix/**
  - tests/software/tabix/tabix/**
```

**Review by drpatelh — APPROVED** [review]

File: tests/software/tabix/tabix/main.nf (hunk 1 above)
Comment (MaxUlysse) [reply tests/software/tabix/tabix/main.nf:7]: Choose just not to include the sam possibility, as it's more common to use bam and their bai index anyway

**Pull request merged**